widget/entry.go

```diff
@@ -1003,6 +994,19 @@ func (e *Entry) textAlign() fyne.TextAlign {
 	return fyne.TextAlignLeading
 }
 
+// textWrap tells the rendering textProvider our wrapping
+func (e *Entry) textWrap() fyne.TextWrap {
+	if e.Wrapping == fyne.TextTruncate {
+		log.Println("Entry does not allow Truncation - how can the user edit text they can't see?")
```

Comment:
I would recommend using fyne.LogError("...", nil) so that developers get more debug info.
Also I think the second half of this sentence is not required, and a little colloquial.

Reply:
Ack